README.markdown

```diff
@@ -56,6 +56,26 @@ always}` as one of the options. You can specify an alternate port with `{port,
 2525}` (default is 25) or you can indicate that the server is listening for SSL
 connections using `{ssl, true}` (port defaults to 465 with this option).
 
+Options
+=======
+    send(Email, Options)
+    send(Email, Options, Callback)
+    send_blocking(Email, Options)
+
+The `send` method variants `send/2, send/3, send_blocking/2` take an `Options` argument.
+`Options` must be a proplist with the following valid values:
+
+  * **relay** the smtp relay, e.g. `"smtp.gmail.com"`
+  * **username** the username of the smtp relay e.g. `"me@gmail.com"`
+  * **password** the password of the smtp relay e.g. `"mypassword"`
+  * **auth** whether the smtp server needs authentication, valid values are `if_available` and `always`, Defaults to `if_available`. If your smtp relay requires authentication set it to `always`
+  * **ssl** whether to connect on 465 in ssl mode, Defaults to `false`
+  * **tls** valid values are `always`, `never`, `if_available`. Most modern smtp relays use tls, so set this to `always`, Defaults to `if_available`
+  * **tls_options** used in ssl:connect, More info at http://erlang.org/doc/man/ssl.html , Defaults to [{versions , ['tlsv1.1', 'tlsv1.2']}], This is merged with options listed at: https://github.com/Vagabond/gen_smtp/blob/master/src/socket.erl#L46 . Any options not present in this list will be ignored.
```

Comment:
From git blame it looks like it was done to avoid incorrect options from being passed: https://github.com/Vagabond/gen_smtp/commit/4b70b227ab5eb0a69d9c9ec27817c9c770606cd9